python/py-dulwich/Portfile

```diff
@@ -32,11 +33,25 @@ if {${name} ne ${subport}} {
 
     patchfiles      patch-archflags.diff
 
+    # Backport of a fix from upstream.
+    # Drop with next release.
+    patchfiles-append \
+                    eb6e78f01a44d738ca3c801d777785910aab5000.patch
+
     depends_lib-append \
-                    port:py${python.version}-setuptools-rust \
+                    port:py${python.version}-setuptools \
```

Comment:
why is `setuptools` and `setuptools-rust` (below) a library dependency? It seems to me that both should only be build dependencies? If so, the explicit `setuptools` can be removed as it's the default backend and thus already added by the `python` PG.

Reply:
Fair enough, I will fix this today.